### PR TITLE
Sorting arrow direction

### PR DESCRIPTION
**Pull request by 1r0n51uv**
Added `asc` prop in order to decide what direction sorting icon should be displayed

### CODE REVIEW

File: src/lib/DataTable/DataTableRouted.tsx [hunk 1]
```diff
@@ -29,14 +29,15 @@ export function DataTableRouted<T, TFilter, TRouteNames>({
   handlers,
   tableClassName,
   tableStyle,
+  asc,
```
Comment (neotrow) [inline src/lib/DataTable/DataTableRouted.tsx:32]: Please add a default value to preserver the existing behavior:

```
asc = true,
```

Comment (neotrow) [reply src/lib/DataTable/DataTableRouted.tsx:32]: I think if you do this you can also remove your stuff again in the test mocks and stories? 
But please double check :-)

File: src/lib/DataTable/DataTableRouted.tsx [hunk 2]
```diff
@@ -29,14 +29,15 @@ export function DataTableRouted<T, TFilter, TRouteNames>({
...
 }: DataTableRoutedProps<T, TFilter, TRouteNames>) {
   const [queryResult, setQueryResult] = useState<TableQueryResult<T>>(data);
   const [filterState, setFilterState] = useState<FilterPageState>({
     currentPage: 1,
     filter: predefinedFilter ?? {},
     itemsPerPage: predefinedItemsPerPage ?? 25,
   });
-  const [orderState, setOrderState] = useState<OrderOption>({ orderBy: columns[0].dataField, asc: true });
+  const [orderState, setOrderState] = useState<OrderOption>({ orderBy: columns[0].dataField, asc: asc });
```
Comment (neotrow) [inline src/lib/DataTable/DataTableRouted.tsx:40]: You can use the shortcut here because the names are the same:
```
  const [orderState, setOrderState] = useState<OrderOption>({ orderBy: columns[0].dataField, asc });
```

Not sure why prettier doesn't complain about it...

**Review by neotrow — CHANGES_REQUESTED** [review]

File: src/lib/DataTable/DataTableRouted.tsx (hunk 1 above)
Comment (1r0n51uv) [reply src/lib/DataTable/DataTableRouted.tsx:32]: Default value added.
I removed only from test mocks. I kept in stories otherwise is not visible in Controls tab below

File: src/lib/DataTable/DataTableInterfaces.ts [hunk 3]
```diff
@@ -51,13 +61,16 @@ export interface DataTableColumnDescription<T> {
   filter?: ColumnFilter;
   enumValues?: EnumValue[];
   dateTimeFormat?: string;
+
```
Comment (neotrow) [inline src/lib/DataTable/DataTableInterfaces.ts:64]: Why did you add this new lines?

File: src/lib/DataTable/DataTableInterfaces.ts [hunk 4]
```diff
@@ -87,6 +100,7 @@ export interface DataTableActions<T> {
 
 export interface DataTableCollapseActions<T> {
   getRows(cell: T): T[];
+
```
Comment (neotrow) [inline src/lib/DataTable/DataTableInterfaces.ts:103]: Same here?

File: src/lib/DataTable/DataTableInterfaces.ts [hunk 5]
```diff
@@ -119,7 +133,9 @@ export interface DeleteAction<T> extends DataTablePredefinedAction<T> {
 
 export interface DataTablePredefinedActionLink<T, TRouteNames> {
   route: TRouteNames;
+
```
Comment (neotrow) [inline src/lib/DataTable/DataTableInterfaces.ts:136]: Same here?

**Review by neotrow — CHANGES_REQUESTED** [review]
Please also update the Changelog.md file. 

If you have questions concerning how to do it please send me a pm in slack. 

Thanks

File: src/lib/DataTable/DataTableInterfaces.ts (hunk 3 above)
Comment (1r0n51uv) [reply src/lib/DataTable/DataTableInterfaces.ts]: Spaced removed

File: src/lib/DataTable/DataTableInterfaces.ts (hunk 4 above)
Comment (1r0n51uv) [reply src/lib/DataTable/DataTableInterfaces.ts]: Spaced removed

File: src/lib/DataTable/DataTableInterfaces.ts (hunk 5 above)
Comment (1r0n51uv) [reply src/lib/DataTable/DataTableInterfaces.ts]: Spaced removed

**Comment by fwermelinger** [comment]
Merging because this has been open for a long time and all the comments from the review were done
